### PR TITLE
Update French Translations

### PR DESCRIPTION
**Pull request by Futuray-pgm**
I think I forgot nothing.
I hope there is no spelling mistake.

### CODE REVIEW

File: Locales/Locale_frFR/Localization.txt [hunk 1]
```diff
@@ -1,90 +1,110 @@
...
-        (key = "AUTH_STATUS_FAIL_VERSION_UPDATE", string = "")
-        (key = "AUTH_STATUS_FAIL_INVALID_SERVER", string = "Impossible de se connecter.")
-        (key = "AUTH_STATUS_FAIL_SUSPENDED", string = "Ce compte à été temporairemenr bloqué.\nVeuillez visiter https://mmo-dev.net/banned.html pour plus d'informations.")
-        (key = "AUTH_STATUS_FAIL_NO_ACCESS", string = "Impossible de se connecter.")
-        (key = "AUTH_STATUS_FAIL_PARENTAL_CONTROL", string = "L'accès à ce compte à été bloqué par le contrôle parental.\nCela peut être modifier dans les paramètres du compte sur https://mmo-dev.net.")
-        (key = "AUTH_STATUS_FAIL_LOCKED", string = "Vous avez bloqué ce compte. Débloquez-le sur https://mmo-dev.net.")
-        (key = "AUTH_STATUS_FAIL_TRIAL", string = "Votre essai gratuit est fini. Allez sur https://mmo-dev.net pour mettre votre compte à jour.")
-        (key = "AUTH_STATUS_FAIL_INTERNAL_ERROR", string = "Erreur interne.")
+	(key = "AUTH_STATUS_SUCCESS", string = "Réussi")
+	(key = "AUTH_STATUS_FAIL_BANNED", string = "Ce compte a été supprimer et n'est desormais plus uttilisable.\nAllez sur https://mmo-dev.net/banned.html pour plus de renseignement.")
```
Comment (Futuray-pgm) [inline Locales/Locale_frFR/Localization.txt:24]: Here, "_a été supprimer_" must be replaced by "_a été supprimé_".

File: Locales/Locale_frFR/Localization.txt [hunk 2]
```diff
@@ -1,90 +1,110 @@
...
+	(key = "AUTH_STATUS_FAIL_WRONG_CREDENTIALS", string = "Les informations entrés sont invalides.\nVeuillez vérifier votre nom et votre mot de passe.\nSi vous avez besoin d'aide ou que vous avez perdu vore mot de passe,\nallez voir sur https://mmo-dev.net.")
+	(key = "AUTH_STATUS_FAIL_ALREADY_ONLINE", string = "Ce compte est déjà connecté. Veuillez vérifier vos paramètres de connections et réessayer.")
+	(key = "AUTH_STATUS_FAIL_NO_TIME", string = "Vos credits gratuits touchent à leurs fin.\nVeuillez en achetez d'auutres pour continuer à jouer.")
+	(key = "AUTH_STATUS_FAIL_DB_BUSY", string = "Impossible de se connecter maintenant. Veuullez réessayer plus tard.")
+	(key = "AUTH_STATUS_FAIL_VERSION_INVALID", string = "Impossible de vérifier votre version de jeu. Cela peut être dût à une corruption des fichiers du jeu ou par un programme tiers .\nAllez sur https://mmo-dev.net pour essayer de trouver une solution à votre problème.")
```
Comment (Futuray-pgm) [inline Locales/Locale_frFR/Localization.txt:29]: Here, "_par_" must be changed with "_à_".

**Review by Futuray-pgm — COMMENTED** [review]
I find some mistake. I'll change them for the next PR

**Review by Kyoril — APPROVED** [review]
Thank you for the effort

**Pull request merged**